laravel/socialite/2015-07-23.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Insecure state generation
+link:      https://github.com/laravel/socialite/pull/91
```

Comment:
Note that just providing some details in the PR (now that it is merged) is ok/sufficient.